src/material.h

```diff
@@ -75,4 +77,13 @@ class material_type
         const mat_burn_data &burn_data( size_t intensity ) const;
 };
 
+namespace materials
```

Comment:
>  If you can give a specific example of where it might be imminently used then that's a valid rebuttal.

material_type::reset(); - this suggests that I reset the single material_type (wrong).
materials::reset(); - this suggests that I reset all the materials (right).